packages/react/use-overlay/src/tests/exit.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import { act, screen, waitFor } from '@testing-library/react';
+import React, { useEffect } from 'react';
+import { useOverlay } from '../useOverlay';
+import { renderWithContext } from './utils';
+
+const closeDuration = 1000;
+
+function OverlayCallerComponent() {
```

Comment:
```suggestion
function TestComponent() {
```

We usually name components like these `TestComponent`s. What about following the convention?